Frameworks/CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -363,10 +359,9 @@ CF_PRIVATE Boolean __CFProcessIsRestricted();
 
 // WINOBJC: The reference platform uses a special compiler option not available to us to 
 // get __CFConstantStringClassReferencePtr filled out with the correct "Class". Since this isn't

```

Comment:
oh, i see. sorry.
I think we should use "WINOBJC" just as liberally in CFBridgeUtilities -- to make sure we know what we "changed" versus what we "moved" :)
We could probably kill this whole comment block in here, and move the commented-out CFEXPORTs to CFBridgeUtilities

---
In reply to: [78864285](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r78864285) [](ancestors = 78864285,78862194)